apps/pie-storybook/stories/pie-avatar.stories.ts

```diff
@@ -25,10 +42,11 @@ const avatarStoryMeta: AvatarStoryMeta = {
 
 export default avatarStoryMeta;
 
-// TODO: remove the eslint-disable rule when props are added
-// eslint-disable-next-line no-empty-pattern
-const Template = ({}: AvatarProps) => html`
-    <pie-avatar></pie-avatar>
+const Template = ({ label, tag }: AvatarProps) => html`
+    <pie-avatar label= "${ifDefined(label)}" tag="${ifDefined(tag)}"></pie-avatar>
```

Comment:
we don't need the space after label=
```suggestion
    <pie-avatar label="${ifDefined(label)}" tag="${ifDefined(tag)}"></pie-avatar>
```